validation/src/test/scala/hmda/validation/rules/lar/quality/Q059Spec.scala

```diff
@@ -0,0 +1,38 @@
+package hmda.validation.rules.lar.quality
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.{ BadValueUtils, LarEditCheckSpec }
+import org.scalacheck.Gen
+
+class Q059Spec extends LarEditCheckSpec with BadValueUtils {
+  property("Lars with loan type equal to 1 must pass") {
```

Comment:
Hm, good question.  There, it feels like it would make things more complicated.  Hm.
I suppose another question is whether at some point, the set of valid values could change, which would (heh) invalidate assumptions elsewhere.  Then again, with the way the rules are written, I suppose that would usually mean updating other rules too.  Or... actually, I just realized I don't actually know what the pattern is, for how the rules are written.  (That question may be easier to discuss in a real-time medium, so I'll stop short for now.)